components/waitwhile/actions/export-customers/export-customers.ts

```diff
@@ -0,0 +1,71 @@
+import waitwhile from "../../app/waitwhile.app";
+import { defineAction } from "@pipedream/types";
+
+export default defineAction({
+  name: "Export Customers",
+  version: "0.0.2",
```

Comment:
Should be version 0.0.1.
```suggestion
  version: "0.0.1",
```